src/javascript/SelectorTypes/Picker/Picker.configs.js

```diff
@@ -108,7 +107,17 @@ export const registerPickerConfig = ceRegistry => {
 
     ceRegistry.add('pickerConfiguration', 'folder', {
         cmp: {
-            picker: contentPicker,
+            picker: {
```

Comment:
why not update directly the source of "contentPicker"

Reply:
Because, we don't want to change the label for every type